### PR TITLE
fixed broken emoji for in "Getting started" section of integration docs

### PR DESCRIPTION
**Pull request by AadityaJain-Dev**
## Changes

- What does this change?
- This PR fixes the issue in integration docs which I reported here https://github.com/withastro/docs/issues/1577
- The emoji is broken in 3rd point of `Getting started` inside integration docs

broken emoji fix

## Testing

All tests are passing

## Docs

Fixes https://github.com/withastro/docs/issues/1577

### CODE REVIEW

File: packages/integrations/lit/README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default {
 To use your first Lit component in Astro, head to our [UI framework documentation][astro-ui-frameworks]. This explains:
 - 📦 how framework components are loaded,
 - 💧 client-side hydration options, and
-- 🪆 opportunities to mix and nest frameworks together
+- 🤝opportunities to mix and nest frameworks together
```
Comment (Yan-Thomas) [inline packages/integrations/lit/README.md:60]:
```suggestion
- 🤝 opportunities to mix and nest frameworks together
```

File: packages/integrations/mdx/README.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ You can [add MDX pages to your project](https://docs.astro.build/en/guides/markd
 To use components in your MDX pages in Astro, head to our [UI framework documentation][astro-ui-frameworks]. You'll explore:
 - 📦 how framework components are loaded,
 - 💧 client-side hydration options, and
-- 🪆 opportunities to mix and nest frameworks together
+- 🤝opportunities to mix and nest frameworks together
```
Comment (Yan-Thomas) [inline packages/integrations/mdx/README.md:81]:
```suggestion
- 🤝 opportunities to mix and nest frameworks together
```

File: packages/integrations/preact/README.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ export default defineConfig({
 To use your first Preact component in Astro, head to our [UI framework documentation][astro-ui-frameworks]. You'll explore:
 - 📦 how framework components are loaded,
 - 💧 client-side hydration options, and
-- 🪆 opportunities to mix and nest frameworks together
+- 🤝opportunities to mix and nest frameworks together
```
Comment (Yan-Thomas) [inline packages/integrations/preact/README.md:73]:
```suggestion
- 🤝 opportunities to mix and nest frameworks together
```

File: packages/integrations/react/README.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ export default {
 To use your first React component in Astro, head to our [UI framework documentation][astro-ui-frameworks]. You'll explore:
 - 📦 how framework components are loaded,
 - 💧 client-side hydration options, and
-- 🪆 opportunities to mix and nest frameworks together
+- 🤝opportunities to mix and nest frameworks together
```
Comment (Yan-Thomas) [inline packages/integrations/react/README.md:60]:
```suggestion
- 🤝 opportunities to mix and nest frameworks together
```

File: packages/integrations/solid/README.md [hunk 5]
```diff
@@ -57,7 +57,7 @@ export default {
 To use your first SolidJS component in Astro, head to our [UI framework documentation][astro-ui-frameworks]. You'll explore:
 - 📦 how framework components are loaded,
 - 💧 client-side hydration options, and
-- 🪆 opportunities to mix and nest frameworks together
+- 🤝opportunities to mix and nest frameworks together
```
Comment (Yan-Thomas) [inline packages/integrations/solid/README.md:60]:
```suggestion
- 🤝 opportunities to mix and nest frameworks together
```

File: packages/integrations/svelte/README.md [hunk 6]
```diff
@@ -57,7 +57,7 @@ export default {
 To use your first Svelte component in Astro, head to our [UI framework documentation][astro-ui-frameworks]. You'll explore:
 - 📦 how framework components are loaded,
 - 💧 client-side hydration options, and
-- 🪆 opportunities to mix and nest frameworks together
+- 🤝opportunities to mix and nest frameworks together
```
Comment (Yan-Thomas) [inline packages/integrations/svelte/README.md:60]:
```suggestion
- 🤝 opportunities to mix and nest frameworks together
```

File: packages/integrations/vue/README.md [hunk 7]
```diff
@@ -57,7 +57,7 @@ export default {
 To use your first Vue component in Astro, head to our [UI framework documentation][astro-ui-frameworks]. You'll explore:
 - 📦 how framework components are loaded,
 - 💧 client-side hydration options, and
-- 🪆 opportunities to mix and nest frameworks together
+- 🤝opportunities to mix and nest frameworks together
```
Comment (Yan-Thomas) [inline packages/integrations/vue/README.md:60]:
```suggestion
- 🤝 opportunities to mix and nest frameworks together
```

**Review by Yan-Thomas — APPROVED** [review]
Thanks, @AadityaJain-Dev! I researched a bit about why this is happening: Windows 10 supports only emojis available till [Emoji 12.0](https://emojipedia.org/emoji-12.0/), being the [Nesting Dolls emoji](https://emojipedia.org/nesting-dolls/) part of Emoji 13.0, which Windows 10 doesn't support and probably never will (the last update was May 2019, adding Emoji 12.0 support). So I agree with changing this for the widely available handshake emoji, considering many people are still using Windows 10.

Note: I see that you ended up removing the whitespace between the emoji and the following text when replicating the PR from the docs repo to here, hope you don't mind me commiting a fix for this.

**Comment by AadityaJain-Dev** [comment]
Thanks for merging my PR

**Pull request merged**